packages/core/src/plugins/PastePlugin.ts

```diff
@@ -0,0 +1,13 @@
+import { Plugin, PluginKey } from '@tiptap/pm/state'
+
+export const PastePlugin = (onPaste: (e: ClipboardEvent) => void) => {
+  return new Plugin({
+    key: new PluginKey('tiptap__paste'),
```

Comment:
Same here.

Reply:
how about `tiptap__pasteEvent` - I'm using the `tiptap__` prefix to make sure to never accidentially conflict with other pluginKeys